services/assistant/v1.js

```diff
@@ -50,10 +52,9 @@ module.exports = function(RED) {
         shape: 'ring',
         text: 'missing payload'
       });
-      node.error('Missing property: msg.payload', msg);
-      return false;
+      return Promise.reject('Missing property: msg.payload');
     }
-    return true;
+    return Promise.resolve();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 8 spaces but found 4.](https://app.codacy.com/manual/BetaWorks-NodeRED-Watson/node-red-node-watson/pullRequest?prid=4882575)